### PR TITLE
[Housekeeping] Moved a UI test to the proper place

### PR DESCRIPTION
**Pull request by kubaflo**
### Description of Change

This test was using an old UI test's approach

### CODE REVIEW

**Review by jsuarezruiz — APPROVED** [review]

File: src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue19127.cs [hunk 1]
```diff
@@ -23,8 +24,8 @@ public void ContentOfFrameShouldChange()
 
 			var textAfterClick = App.FindElement("label2").GetText();
 
-			Assert.AreEqual(textBeforeClick, "Camera is Disabled");
-			Assert.AreEqual(textAfterClick, "Camera is Enabled");
+			Assert.That(textBeforeClick!, Is.EqualTo("Camera is Disabled"));
```
Comment (jsuarezruiz) [inline src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue19127.cs:27]: Failing on Android and Catalyst:
image

**Review by jsuarezruiz — CHANGES_REQUESTED** [review]

File: src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue19127.cs (hunk 1 above)
Comment (kubaflo) [reply src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue19127.cs:27]: Can wy try again?

**Review by jsuarezruiz — APPROVED** [review]

**Pull request merged**